tests/testthat/test-api-endpoints.R

```diff
@@ -577,3 +577,18 @@ test_that("run metadata can get name from config", {
     changelog_types = NULL
   ))
 })
+
+test_that("git branches endpoint", {
+  path <- orderly_prepare_orderly_git_example()
+  branch_data <- data.frame(
+    name = c("master", "dev-branch"),
+    last_commit = c("Mon Jun 1 16:00:41 2020 +0100",
```

Comment:
Can we make this `last_commit_time` and also add `last_commit_age` which would be an integer in seconds? That will avoid a lot of time zone confusion.

Also this time should be ISO formatted